src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import { useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+import { Tabs } from '../components/Tabs';
+
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+const TabsPage = () => {
+  const { tabId } = useParams();
```

Comment:
The useParams hook should be typed to avoid possible type issues. You can add a type for tabId like this: `const { tabId } = useParams<{ tabId: string }>();`

Reply:
Fixed